src/sql/base/browser/ui/taskbar/overflowActionbarStyles.ts

```diff
@@ -1,43 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the Source EULA. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-// eslint-disable-next-line code-import-patterns
```

Comment:
note - we missed this in the previous PR but we should never disable code import patterns like this unless we have a good reason and are sure it isn't going to cause problems (and that should be an extremely rare scenario)